app/views/parsers/_form.html.erb

```diff
@@ -2,8 +2,12 @@
   <% if can? :update, @parser %>
     <%= f.text_area :content, class: "code-editor read" %>
 
+    <% if @parser.errors.any? %>
+      <span class="error">The parser version message is required.</span>
+    <% end %>
```

Comment:
The error display is already handled in the `custom_form_builder.rb`. Was it not showing up?

Reply:
No it wasn't showing up. It could be because the version is a relation of the parser?

Reply:
Could be, the error has to be `errors[:message]` or `errors[:message_id]`. Makes me think that you are displaying `The parser version message is required.` if there's any error. Could you display in just if that's the error you have? I believe you can have other kind of errors on the parser.

Reply:
Yeah sure, the other errors for the parser version are pre populated so it should be safe. I'll see why the custom form errors aren't coming through